DeepSpeech.py

```diff
@@ -771,6 +771,20 @@ def export():
     from tensorflow.python.framework.ops import Tensor, Operation
 
     inputs, outputs, _ = create_inference_graph(batch_size=FLAGS.export_batch_size, n_steps=FLAGS.n_steps, tflite=FLAGS.export_tflite)
+
+    graph_version = int(file_relative_read('GRAPH_VERSION').strip())
+    assert graph_version > 0
+
+    # Reshape with dimension [1] required to avoid this error:
```

Comment:
FWIW you should also be able to just do `tf.constant([value])`

Reply:
Yeah, looks like from all the solutions I tried, I forgot to test this one, or it was failing due to something else. Now works, so let's do that.